web/static/css/_quex-simulation.scss

```diff
@@ -3,6 +3,63 @@ $atMessageBackground: color("grey", "darken-4");
 $chatWindowBodyBackground: #999999;
 $chatBorderBackground: color("grey", "darken-3");
 
+// every simulation component heightens 8.5 simulation steps
+// the simulation step height depends on the screen size
+$simulationHeight: calc(74px * 8.5);
+$simulationHeightMediumAndDown: calc(72px * 8.5);
+
+// chatBodyHeight = chatTotalHeight - (chatHeaderHeight + chatFooterHeight)
+$chatWindowHeaderHeight: 90px;
+$chatWindowBodyHeight: calc((74px * 8.5) - (90px + 81px));
```

Comment:
Maybe you are not able to mention the sass variables in a calc because they should be interpolated with `#{}`

```
$a: 4px;
$b: calc(#{$a} + 7px);
  
div {
  height: $b;
```

But in this case, since the input of the calculation are all px and not mixed with % you can avoid using css calc and let sass compute the value of the expression directly.

```
$a: 4px;
$b: $a + 7px;
```

Reply:
Yes! I didn't know any of this! Please take a look again! Thanks!!